Kuni 6. ülesandeni tehtud Martin Toomas Kripsaar

martkrip opened the pull request:
Praegusel seisul on mul 1. ja 2. ülesanne tehtud

martkrip committed:
1. ja 2. ülesanne tehtud Martin Toomas Kripsaar

Praegusel seisul on mul 1. ja 2. ülesanne tehtud

taurikirsipuu commented:
Teine pole tehtud, sest tulemusi peavad olema modali sees selleks. Praegu midagi hinnata ei saanud veel.

martkrip committed:
2. ülesanne tehtud (päriselt)

martkrip committed:
3. ja 4. ülesanne tehtud

martkrip committed:
5. ja 6. ülesanne tehtud

Tegin ühe mobiiliversiooni ära. Kujundasin veidike veebilehe läbi CSSi. Muutsin värve vastavalt minu päritolule (Ukraina-eestlane). Loodan, et sellest saab 12 punkti. Kui mitte, siis täiendan juurde.

martkrip commented:
Nii, ma tegin kuni 6. ülesandeni. Kas võiksid palun üle vaadata ja öelda, kas sellest piisab, et 12 punkti saada? Kui ei piisa, siis hea meelega teen juurde.
